docs/UserGuide.adoc

```diff
@@ -59,6 +59,8 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 
 The timetable content is locked, and it requires a user to either login with a pre-existing account, or register a new one.
 
+NOTE: The application have a default user with username: *test* and password: *test*
```

Comment:
has*

though i don't think we actually want to advertise this to people...... just make them register their own account instead of telling them this thing exists?

Reply:
Its for a convenience thing. They would not want to create two accounts to login cause they would have to type out all the whole field

Reply:
hmm fair enough i guess